opta/module_processors/datadog.py

```diff
@@ -38,7 +38,7 @@ def process(self, module_idx: int) -> None:
             field_selector=f"metadata.name={self.layer.name}"
         )
         if len(namespaces.items) == 0:
-            self.v1.create_namespace(
+            self.v1.create_namespace_if_not_exists(
```

Comment:
Is it possible to use the methods from `kubernetes.py` here instead of calling k8s directly?

Reply:
oh crap my bad-- did not mean to update this one